### PR TITLE
Fuel and Plants Cross Compat

### PR DESCRIPTION
**Pull request by MuteTiefling**
Added various fuels to the Thermal Compression Dynamo.

Added missing Farmer's Delight and Simple Farming plants to the Phytogenic Insolator. Don't worry, left out the grapes :D

Added a few gems to the Lapidary Dynamo as well for fun. Brought the orignal recipes over as well to simplify potential future tweaks. Both of these dynamo scripts have a Multiplier as well so you can easily make a blanket change to all fuel types if needed for balancing.

Lastly, minor update to the Storage tags, forgot some inventories from Farmer's Delight.

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_farmersdelight_crops.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		  result: [
+			{
+			  item: recipe.outputs[0],
+			  chance: recipe.chances[0]
+			},
+			{
+			  item: recipe.outputs[1],
+			  chance: recipe.chances[1]
+			},
+			{
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_farmersdelight_crops.js:49]: Does this script work? I see you're accessing indexes in the arrays that don't exist.

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_farmersdelight_crops.js:49]: Woops, placed the comment incorrectly :P do all the insolator recipes work?

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_farmersdelight_crops.js:49]: It does work, though I don't know if this is bad practice. It seems to revert to minecraft:air in the absence of a valid item, and 0 in the absence of a valid float. 

I was a little surprised by it too, so I double checked that all of the recipes were showing up in JEI.

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_farmersdelight_crops.js:49]: Huh alright, if it works it works :D

**Comment by NillerMedDild** [comment]
Thank you kindly!

**Comment by MuteTiefling** [comment]
Well, I said they work and then immediately found some errors in a few. New PR incoming.

**Pull request merged**